cmake/Dependencies.cmake

```diff
@@ -1305,7 +1302,7 @@ endif()
 
 if(USE_DISTRIBUTED AND USE_TENSORPIPE)
   if(MSVC)
-    message(WARNING "Tensorpipe cannot be used on Windows.")
+    message(FATAL_ERROR "Tensorpipe cannot be used on Windows.")
```

Comment:
does this mean we should by default set `USE_TENSORPIPE` to `OFF` in `/CMakeLists.txt` if it is Windows?

Reply:
yeah, that is a better way.